doc/pages/user/030-manage_interfaces.md

```diff
@@ -72,52 +131,108 @@ _Sample Response_
 
 ## Installing/Updating an interface
 
-Interfaces are supposed to change over time, and are dynamic. As such, they can be installed and updated. Interface installation means adding either a whole new interface (as in: an interface with a new name), or a new major version of an already known interface. Interface update means updating a specific, existing interface name/major version with a new minor version.
+Interfaces are supposed to change over time, and are dynamic. As such, they can be installed and updated. Interface installation means adding either a whole new Interface (as in: an Interface with a new name), or a new major version of an already known Interface. Interface update means updating a specific, existing interface name/major version with a new minor version.
 
-### Installation
+When designing interfaces, it is strongly advised to use Astarte Interface Editor. The Editor is embedded into any Astarte Dashboard installation but, in case your Astarte installation does not provide you with a Dashboard, you can use [Astarte Interface Editor public online instance](https://interfaces-editor.astarte-platform.org). Use it to write and validate your definitions, and install the resulting JSON file through either `astartectl` or Realm Management APIs.
 
-To install a new interface, `POST` its JSON body to the `/interfaces` endpoint of the Realm encapsulated in a `data` object, like in the following example:
+### Synchronizing interfaces using astartectl
 
+`astartectl` provides a handy `sync` command that, given a list of Interface files, will synchronize the state of the Astarte Realm with your local interfaces. It is handy in those cases where your Realm has several interfaces, and you're storing Interfaces in a common place, such as a Git Repository - this is the average case for Astarte-based applications/clouds.
+
+Assuming you have a set of Interface files in your folder all with the `.json` extension, invoking `astartectl realm-management interfaces sync` will result in something like this:
+
+```bash
+$ astartectl realm-management interfaces sync *.json
+Will install interface com.my.Interface1 version 0.2
+Will install interface com.my.Interface2 version 1.1
+Will update interface com.my.Interface3 to version 1.4
+
+Do you want to continue? [y/n] y
+Interface com.my.Interface1 installed successfully
+Interface com.my.Interface2 installed successfully
+Interface com.my.Interface3 updated successfully to version 1.4
 ```
+
+After invocation, your Astarte Realm will be up to date with all Interfaces in your local directory.
+
+*Note: `astartectl realm-management interfaces sync` currently synchronizes Interfaces only from your local machine to the Realm, and not the other way round. In case the Realm has a more recent version of an interface compared to your local files, or it has some interfaces which are not referenced by your local files, no action will be taken.*
```

Comment:
Let's add `<!-- TODO: blah blah, issue number here -->`. Using comments with TODO will allow us to update documentation as soon as we update astartectl.

Reply:
I don't think that command will ever be updated, at least in the short term, and that feature isn't planned. The note is ok as it is and it requires no TODOs.